HARK/ConsumptionSaving/ConsPortfolioModel.py

```diff
@@ -173,6 +172,8 @@ def __init__(self, verbose=False, quiet=False, **kwds):
         params.update(kwds)
         kwds = params
 
+        self.PortfolioBool = True
```

Comment:
Where is the documentation for this variable? It could go in the docstring.

Why is this variable named in CamelCase rather than snake_case?

Reply:
I tried to name it in a similar pattern to `CubicBool` and `vFuncBool`